cartography/intel/aws/ec2.py

```diff
@@ -606,3 +611,259 @@ def sync_load_balancers(session, boto3_session, regions, current_aws_account_id,
         data = get_loadbalancer_data(boto3_session, region)
         load_load_balancers(session, data, region, current_aws_account_id, aws_update_tag)
     cleanup_load_balancers(session, common_job_parameters)
+
+
+def get_ec2_vpcs(session):
```

Comment:
Please reorganize these methods to be consistent with the rest of the file -- all get_ methods in one block, then all load_ methods, then all cleanup_ methods, then all sync_ methods.